experiments/calibration/shared/observation_data_global.jl

```diff
@@ -0,0 +1,26 @@
+import ClimaAnalysis
+using ClimaLand
+using ClimaLand.ClimaArtifacts
+
+filename = joinpath(
+    ClimaLand.Artifacts.era5_surface_data2008_path(),
+    "era5_monthly_surface_fluxes_200801-200812.nc",
+)
+
+era5_lhf = ClimaAnalysis.OutputVar(filename, "mslhf")
+era5_shf = ClimaAnalysis.OutputVar(filename, "msshf")
+era5_lwu = ClimaAnalysis.OutputVar(filename, "msuwlwrf")
+era5_swu = ClimaAnalysis.OutputVar(filename, "msuwswrf")
+
+observations_vecs = []
+for var in [era5_lhf, era5_shf, era5_lwu, era5_swu]
```

Comment:
If the following implies the loss function is based on a global average, e.g., terms like gavg(lhf_obs) - gavg(lhf_sim), it's not right. The averages appearing in the loss functions should be spatially resolved time averages, e.g., seasonal_average(lhf_obs) - seasonal_average(lhf_sim). The global mean does not contain sufficient information for meaningful model calibration.